Sources/tart/VM.swift

```diff
@@ -219,21 +219,26 @@ class VM: NSObject, VZVirtualMachineDelegate, ObservableObject {
   func run(_ recovery: Bool) async throws {
     try network.run(sema)
 
-    DispatchQueue.main.sync {
+    let task = DispatchQueue.main.sync {
```

Comment:
What do you think of renaming it to something like `vmStartTask` or `startTask`?

Reply:
See https://github.com/cirruslabs/tart/pull/343/commits/213d5cc4d98b4f279080c677ee8b9867bd3d1fc2.